client/my-sites/stats/site.jsx

```diff
@@ -581,6 +581,7 @@ function StatsBody( { siteId, chartTab = 'views', date, context, isInternal, ...
 						showArrows={ ! wpcomShowUpsell }
 						slug={ slug }
 						dateRange={ customChartRange }
+						appliedShortcut={ appliedShortcut }
```

Comment:
This might not be necessary as the dateRange should already have the shortcut ID.

From line 388 above:

```
customChartRange.shortcutId = appliedShortcut.id;
```

Reply:
Good catch! Adressed in 079fc8c8d60